test/Project.toml

```
@@ -13,4 +13,4 @@ SumOfSquares = "4b9e565b-77fc-50a5-a571-1244f986bda1"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
-DynamicPolynomials = "0.5"
+DynamicPolynomials = "0.6"
```

```suggestion
DynamicPolynomials = "0.5, 0.6"
```
This should fix the build error on Julia v1.6.

---

Thanks!

---

Huh, now the older version is picked up by the v1.12 build too :thinking:
I let you figure it out, sorry :sweat_smile: